exp/views/responses.py

```diff
@@ -1108,6 +1160,62 @@ def get(self, request, *args, **kwargs):
         set_content_disposition(response, filename)
         return response
 
+class StudyResponsesFrameDataPsychDS(ResponseDownloadMixin, generic.list.ListView):
+    """Hitting this URL downloads a ZIP file in Psych-DS formatting with frame data from one response per file in CSV format"""
+
+    # TODO: with large files / many responses generation can take a while. Should generate asynchronously along
+    # with the data dict.
+    def render_to_response(self, context, **response_kwargs):
```

Comment:
Can we have an issue to go with this todo?

Reply:
@bleonar5 is there an issue for this now?